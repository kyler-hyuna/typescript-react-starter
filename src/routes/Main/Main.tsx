import React from 'react'
import { Link } from 'react-router-dom'

const Main = () => <Link to="/next">Go to Next</Link>

export default Main
