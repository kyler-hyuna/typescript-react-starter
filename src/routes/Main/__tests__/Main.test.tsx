import React from 'react'
import { MemoryRouter } from 'react-router-dom'
import { create } from 'react-test-renderer'
import Main from '../Main'

describe('Main', () => {
  it('should work', () => {
    const tree = create(
      <MemoryRouter>
        <Main />
      </MemoryRouter>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
