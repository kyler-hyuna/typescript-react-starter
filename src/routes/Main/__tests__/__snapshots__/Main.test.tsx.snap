// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should work 1`] = `
<a
  href="/next"
  onClick={[Function]}
>
  Go to Next
</a>
`;
